.github/workflows/ci.yml

```diff
@@ -420,15 +420,26 @@ jobs:
     - name: Install dependencies
       run: |
         brew update
-        brew install libmad libpng jpeg-turbo sdl2
+        brew install libmad libpng jpeg-turbo
+    - name: Add support files
```

Comment:
We should add a step that caches the downloaded SDL framework to this and the build_macos jobs to cut down on build times.

Would you like to do that or would you prefer it someone else did?

Reply:
Pushed 0510832 which should handle this.